packages/codelabs/src/intermediate/lit-html.md

```diff
@@ -74,79 +74,106 @@ You should already know how to create a web component using `LitElement`. Go ahe
 <!DOCTYPE html>
 <html>
   <body>
-    <news-app></news-app>
+    <hn-comment-app></hn-comment-app>
 
     <script type="module">
       import { LitElement, html } from 'https://unpkg.com/lit-element?module';
 
-      class NewsApp extends LitElement {
+      class HackerNewsCommentApp extends LitElement {
         render() {
           return html`
-            My news app
+            My comment overview app
           `;
         }
       }
 
-      customElements.define('news-app', NewsApp);
+      customElements.define('hn-comment-app', HackerNewsCommentApp);
     </script>
   </body>
 </html>
 ```
 
 </details>
 
-## Google news API
+## External API
 
-For this codelab, we will be fetching live data from the google news API. You will need to retrieve an API key first:
+For this codelab, we will be fetching live data from the HackerNews API. To retrieve an
```

Comment:
Why is there a hard line break here? Its in some other places too

Reply:
I break my code around like 80, just a habbit i started. I will update these lines to not wrap.